tests/test_recipe.py

```diff
@@ -146,6 +156,26 @@ def test_download_url_is_set(self):
                 'https://www.python.org/ftp/python/3.7.4/Python-3.7.4.tgz')]
         assert m_touch.call_count == 1
 
+    def test_hashes(self):
+        """
+        Verifies digest hashes are enforced.
+        """
```

Comment:
Could be one-line docstring https://www.python.org/dev/peps/pep-0257/#one-line-docstrings